tempus_fugit_minecraft/player.py

```diff
@@ -16,12 +16,13 @@ def __init__(self) -> None:
         @see [Issue#67](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/67)
         """
         # About the height of a block.
-        self.MAX_JUMP_HEIGHT = 1.0
-        self.MAX_FALL_SPEED = 50
-        self.FLYING_SPEED = 15
-        self.GRAVITY = 20.0
-        self.MAX_SPEAD = 15
-        self.MIN_SPEAD = 5
+        self.MAX_JUMP_HEIGHT_IN_BLOCKS = 1.0
+        self.PLAYER_HEIGHT_IN_BLOCKS = 2
+        self.MAX_FALL_SPEED_IN_BLOCKS = 50
+        self.FLYING_SPEED_IN_BLOCKS_PER_SECOND = 15
+        self.GRAVITY_IN_BLOCKS_PER_SECOND_SQUARED = 20.0
```

Comment:
could you find a relation between this value and real gravity?

Reply:
The value is derived from the legacy code and I only changed the name of it. If you feel the value should be different, please file an issue and we can discuss it there.